web-ng/src/app/shared/models/project.model.ts

```diff
@@ -31,4 +44,14 @@ export class Project {
   getLayer(layerId: string): Layer | undefined {
     return this.layers.get(layerId);
   }
+
+  isEmpty() {
```

Comment:
It might not be clear why we have `EMPTY` and `isEmpty` from the name - maybe we can call these `UNSAVED_NEW` and `isUnsavedNew()`, respectively?

Reply:
Done.